iml-services/iml-device/src/main.rs

```diff
@@ -165,9 +165,11 @@ async fn update_client_mounts(
                 mountpoint = Null,
                 state = 'unmounted',
                 state_modified_at = now()
-            WHERE filesystem != ALL ($1)
+            WHERE host_id = $1
```

Comment:
This was lost in https://github.com/whamcloud/integrated-manager-for-lustre/pull/2025, right?

Reply:
Yes